api/tag/models.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import (Column, String, Table, Integer, ForeignKey)
+from sqlalchemy.orm import relationship
+# from sqlalchemy.schema import Sequence
+
+from helpers.database import Base
+from utilities.utility import Utility, validate_empty_fields
+
+room_tags = Table(
```

Comment:
Is there a reason why you have this variable here to define the relationship instead of defining the relationship directly in the Tag class?

Reply:
Yes, I followed the implementation as described in the sqlalchemy documentation here: [https://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#many-to-many](https://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#many-to-many).
However, I now made a small change to use the variable name (room_tags) in the secondary argument of the relationship instead of room_tags as a string.
It should be noted that both the variable name and the string name seems to work just fine.